examples/pxScene2d/src/shell.js

```diff
@@ -199,12 +193,9 @@ if (false)
 
   scene.root.on("onPreChar", function(e)
   {
-    console.log("in onchar");
 	  var c = e.charCode;
-    console.log("onChar:", c);
 	  // TODO eating some "undesired" chars for now... need to redo this
```

Comment:
For the time being, let's leave the key prints in place as they are useful for debugging.